perf/lua/1mops_write.lua

```diff
@@ -28,12 +28,25 @@ local params = require('internal.argparse').parse(arg, {
 
 local test_dir = fio.tempdir()
 
+local function rmtree(s)
+    if (fio.path.is_file(s) or fio.path.is_link(s)) then
+        fio.unlink(s)
+        return
+    end
+    if fio.path.is_dir(s) then
+        for _,i in pairs(fio.listdir(s)) do
+            rmtree(s..'/'..i)
+        end
+        fio.rmdir(s)
+    end
+end
+
 local function exit(res, details)
     if (details ~= nil) then
         print(details)
     end
     if test_dir ~= nil then
-        fio.rmdir(test_dir)
+        rmtree(test_dir)
```

Comment:
There's `fio.rmtree()` and it seems to be doing the same thing: https://www.tarantool.io/en/doc/latest/reference/reference_lua/fio/#lua-function.fio.rmtree
Can you use it here?

Reply:
Ok, I see now. Thanks!